app/src/main/res/values/strings.xml

```diff
@@ -302,4 +302,25 @@
     <string name="dismiss_temporary">Not now</string>
     <string name="update_available">New update available</string>
     <string name="update_available_description">A new version (%s) is available for download.</string>
+
+    <string name="installer_result_failed">Failed to install</string>
+    <string name="installer_result_cancelled">Cancelled</string>
+    <string name="installer_result_blocked">Blocked</string>
+    <string name="installer_result_conflict">Conflict</string>
+    <string name="installer_result_incompatible">Incompatible</string>
+    <string name="installer_result_invalid">Invalid</string>
+    <string name="installer_result_storage_issue">Storage issue</string>
+    <string name="installer_result_timeout">Timeout</string>
+    <string name="installer_result_success">Success</string>
+    <string name="installer_result_unknown">Unknown</string>
+
+    <string name="installer_result_failed_description">The installation failed for an unknown reason.</string>
+    <string name="installer_result_aborted_description">The installation was manually cancelled.</string>
+    <string name="installer_result_blocked_description">The installation was blocked.</string>
+    <string name="installer_result_conflict_description">An existing installation of the app prevents the installation. Uninstall the app and try again?</string>
+    <string name="installer_result_incompatible_description">The app is incompatible with this device.</string>
+    <string name="installer_result_invalid_description">The app is invalid.</string>
```

Comment:
I think it would be helpful to append `Uninstall the existing app and try again?` to this description as well, as this error is most often seen when the user tries downgrading. When downgrading, the package installer will prioritize this error over the conflict error, even when the signatures don't match.